plugins-compat-tester/pom.xml

```diff
@@ -43,6 +43,11 @@
   
   <dependencies>
     <dependency>
+      <groupId>org.jenkins-ci</groupId>
```

Comment:
Any issue with the one already included through the core dependency?

Reply:
I had some unexplained errors in it when working when `.*` version components. Since @recena overhauled the algorithm a while back I figured we may as well use the current stuff.

Reply:
I was just wondering as we faced some similar issues in the past too.